src/components/split-pane/SplitPane.tsx

```diff
@@ -111,48 +134,42 @@ export function SplitPane(props: SplitPaneProps) {
     sizeType,
     onSizeChange(value) {
       touch();
-      setSize(value);
+      const serialized = serializeSize(value);
+      setSize(serialized);
+      onSizeChange?.(serialized);
     },
     onResize,
   });
 
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore Module exists.
+  // @ts-expect-error I do not understand why there is a TS error here.
```

Comment:
@targos could you have a look?

I don't understand why there is an error.

Reply:
I don't understand either, but this is why I used `@ts-ignore` in this case.